docs/schema.md

```diff
@@ -693,3 +693,19 @@ indicates whether Istio is enabled on the cluster.
 - `OPTIONAL`: The app works with Istio but does not require it.
 - `REQUIRED`: The app requires Istio to work properly.
 - `UNSUPPORTED`: The app does not support Istio.
+
+## form
+
+### help widget
+
+You can add a blob of basic HTML text to the top of the configuration form
```

Comment:
It would help to include more supported HTML elements in the example (or list what is or isn't supported - I don't know exactly what this list is), as partners have asked in the past.